Sources/Responses/Webhook.swift

```diff
@@ -202,9 +202,9 @@ public class Webhook: BoxModel {
             case .taskAssignmentUpdated:
                 return "TASK_ASSIGNMENT.UPDATED"
             case .metadataInstanceCreated:
-                return "METEDATA_INSTANCE.CREATED"
+                return "METADATA_INSTANCE.CREATED"
             case .metadataInstanceUpdated:
-                return "METEDATA_INSTANCE.UPDATED"
+                return "METADATA_INSTANCE.UPDATED"
```

Comment:
Just double-checking: not a breaking change?

Reply:
Unfortunately, but because of this typo, the backend was receiving invalid values that it can not recognize

It's not a breaking change because it doesn't affect the existing API in any way.